README.md

```diff
@@ -1,22 +1,123 @@
-[![Build Status](https://travis-ci.org/IBM-Swift/generator-swiftserver.svg?branch=master)](https://travis-ci.org/IBM-Swift/generator-swiftserver)
-[![codecov](https://codecov.io/gh/IBM-Swift/generator-swiftserver/branch/master/graph/badge.svg)](https://codecov.io/gh/IBM-Swift/generator-swiftserver)
-
 # generator-swiftserver
 
+[![Build Status](https://travis-ci.org/IBM-Swift/generator-swiftserver.svg?branch=master)](https://travis-ci.org/IBM-Swift/generator-swiftserver)
+[![codecov](https://codecov.io/gh/IBM-Swift/generator-swiftserver/branch/master/graph/badge.svg)](https://codecov.io/gh/IBM-Swift/generator-swiftserver)
+[![Version](https://img.shields.io/npm/v/generator-swiftserver.svg)][url-npm]
+[![DownloadsMonthly](https://img.shields.io/npm/dm/generator-swiftserver.svg)][url-npm]
+[![DownloadsTotal](https://img.shields.io/npm/dt/generator-swiftserver.svg)][url-npm]
+[![License](https://img.shields.io/npm/l/generator-swiftserver.svg)][url-npm]
 [![Greenkeeper badge](https://badges.greenkeeper.io/IBM-Swift/generator-swiftserver.svg)](https://greenkeeper.io/)
 
+[url-npm]: https://www.npmjs.com/package/generator-swiftserver
+
 This module is a [Yeoman](http://yeoman.io) generator for creating REST webservices based on the [Kitura web framework](http://kitura.io) with the [Swift](https://swift.org/) language.
 
 This generator and the Yeoman library runs on Node.js and generates a Swift 4 application.
 
-## Installation
+## Contents
+1. [Prerequisites](#prerequisites)
+1. [Installation](#installation)
+1. [Usage](#usage)
+    - [Options](#options)
+    - [Run](#run)
+1. [Generated Artifacts](#generated-artifacts)
+1. [Development](#development)
+1. [Testing](#testing)
+1. [Contributing](#contributing)
+
+## Prerequisites
 To use this module, you will need Node.js and Swift 4 installed on your Linux or macOS system. You can get Node.js from https://nodejs.org and Swift 4 from https://swift.org/download.
 
-You will also need the Yeoman command line utility [yo](https://github.com/yeoman/yo) installed in your global Node.js module directory.  
-To install yo, use: `npm install -g yo`  
-To install this generator, use: `npm install -g git+https://github.com/IBM-Swift/generator-swiftserver`
+You will also need the Yeoman command line utility [yo](https://github.com/yeoman/yo) installed in your global Node.js module directory.
+ 
+To install `yo`, run:
+
+```bash
+npm install -g yo
+```
+
+## Installation
+To install generator-swiftserver, run:
+
+```bash
+npm install -g git+https://github.com/IBM-Swift/generator-swiftserver
```

Comment:
Is the standard convention here to use the git url or the npm module name?

Reply:
Good point.. I forgot to ask about this... Would you happen to know why the current README uses git url instead of just `npm install -g generator-swiftserver`? If there is no specific reason, then I agree that we should follow standard convention and use the module name.